xc7/utils/vivado_output_timing.py

```diff
@@ -0,0 +1,32 @@
+""" Utility for generating TCL script to output timing information from a
+design checkpoint.
+"""
+import argparse
+
+
+def create_runme(f_out, args):
```

Comment:
nit: name doesn't reflect function

```suggestion
def create_output_timing(f_out, args):
```

Reply:
Fixed